apps/server/src/controllers/service.js

```diff
@@ -0,0 +1,33 @@
+import { getLogInfos } from '../utils/logger.js'
+import { send200, send500 } from '../utils/response.js'
+
+import {
+  getRoleByUserIdAndProjectId,
+} from '../models/queries/users-projects-queries.js'
+
+// POST
+export const checkServiceHealthController = async (req, res) => {
+  const requestorId = req.session?.user?.id
+  const projectId = req.params.projectId
+  const data = req.body
+
+  try {
+    const role = await getRoleByUserIdAndProjectId(requestorId, projectId)
+    if (!role) throw new Error('Vous n\'êtes pas membre du projet')
+
+    const serviceRes = await fetch(data.to)
```

Comment:
TODO : fetch `healthz` de chaque service

Reply:
@this-is-tobi : la plupart des healthcheck te donnent l'état de tout le site il semblerait. Par ex : 
sonarqube ça va être `<sonarUrl>.io/api/system/health` => l'état du sonarqube interne, si le site tourne ça n'empêche que la page du projet pourrait être en 404. Est-ce qu'il ne vaut pas mieux rester sur fetch du coup ? 
ça dépend de ce que l'on entend par vérifier l'état du service

Reply:
Ah moi je pensais à l'état du Gitlab, du Sonar etc de manière générale, si ta page est en `404` c'est que ton projet n'existe pas dans le service (si les outils ont bien fait leur statuts http). On peut en parler de vive voix 📣

Reply:
Ok parfait on reste plutôt sur un fetch du projet dans le service alors !